Box86Launcher crashes after launching explorer or cmd on Android 12, Samsung S10e

As title says - worth mentioning that I own Exynos-based phone. Where can I find logs to repost them here?

Sorry for being late here. I tested it on my Android 12 phone. I can confirm that the app crashes when trying to launch explorer or cmd. Here's the crash report: 
```
FATAL EXCEPTION: main
Process: com.allplatform.box86, PID: 20292
java.lang.RuntimeException: Unable to create service [com.termux.app](http://com.termux.app/).TermuxService: java.lang.IllegalArgumentException: com.allplatform.box86: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
at [android.app](http://android.app/).ActivityThread.handleCreateService([ActivityThread.java:4500](http://activitythread.java:4500/))
at [android.app](http://android.app/).ActivityThread.access1700([ActivityThread.java:247](http://activitythread.java:247/))H.handleMessage([ActivityThread.java:2072](http://activitythread.java:2072/))
at android.os.Handler.dispatchMessage([Handler.java:106](http://handler.java:106/))
at android.os.Looper.loopOnce([Looper.java:201](http://looper.java:201/))
at android.os.Looper.loop([Looper.java:288](http://looper.java:288/))
at [android.app](http://android.app/).ActivityThread.main([ActivityThread.java:7839](http://activitythread.java:7839/))
at java.lang.reflect.Method.invoke(Native Method)
at [com.android](http://com.android/).internal.os.RuntimeInit$MethodAndArgsCaller.run([RuntimeInit.java:550](http://runtimeinit.java:550/))
at [com.android](http://com.android/).internal.os.ZygoteInit.main([ZygoteInit.java:1003](http://zygoteinit.java:1003/))
Caused by: java.lang.IllegalArgumentException: com.allplatform.box86: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
at [android.app](http://android.app/).PendingIntent.checkFlags([PendingIntent.java:375](http://pendingintent.java:375/))
at [android.app](http://android.app/).PendingIntent.getActivityAsUser([PendingIntent.java:458](http://pendingintent.java:458/))
at [android.app](http://android.app/).PendingIntent.getActivity([PendingIntent.java:444](http://pendingintent.java:444/))
at [android.app](http://android.app/).PendingIntent.getActivity([PendingIntent.java:408](http://pendingintent.java:408/))
at [com.termux.app](http://com.termux.app/).TermuxService.buildNotification([TermuxService.java:207](http://termuxservice.java:207/))
at [com.termux.app](http://com.termux.app/).TermuxService.onCreate([TermuxService.java:189](http://termuxservice.java:189/))
at [android.app](http://android.app/).ActivityThread.handleCreateService([ActivityThread.java:4487](http://activitythread.java:4487/))
... 9 more
```

Fixing this error is quite easy. I'll make a pull request for that.

--- a/app/src/main/java/com/termux/app/TermuxService.java
+++ b/app/src/main/java/com/termux/app/TermuxService.java
@@ -204,7 +204,7 @@ public final class TermuxService extends Service implements SessionChangedCallba
         // PendingIntent#getActivity(): "Note that the activity will be started outside of the context of an existing
         // activity, so you must use the Intent.FLAG_ACTIVITY_NEW_TASK launch flag in the Intent":
         notifyIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0, notifyIntent, 0);
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0, notifyIntent, PendingIntent.FLAG_IMMUTABLE);
 
         int sessionCount = mTerminalSessions.size();
         int taskCount = mBackgroundTasks.size();

--- a/app/src/main/java/x/org/server/DummyService.java
+++ b/app/src/main/java/x/org/server/DummyService.java
@@ -74,7 +74,7 @@ public class DummyService extends Service
 			.setOngoing(true)
 			.build();
 		PendingIntent killIntent = PendingIntent.getService(this, 5, new Intent(Intent.ACTION_DELETE, null, this, DummyService.class), PendingIntent.FLAG_CANCEL_CURRENT);
-		PendingIntent showIntent = PendingIntent.getActivity(this, 0, new Intent("", null, this, MainActivity.class), PendingIntent.FLAG_CANCEL_CURRENT);
+		PendingIntent showIntent = PendingIntent.getActivity(this, 0, new Intent("", null, this, MainActivity.class), PendingIntent.FLAG_CANCEL_CURRENT | PendingIntent.FLAG_IMMUTABLE);
 		ntf.deleteIntent = killIntent;
 		RemoteViews view = new RemoteViews(getPackageName(), R.layout.notification);
 		view.setCharSequence(R.id.notificationText, "setText", getString(R.string.notification_app_is_running, getString(getApplicationInfo().labelRes)));

